src/Avalonia.Controls/Automation/Peers/ControlAutomationPeer.cs

```diff
@@ -117,15 +117,18 @@ protected override IReadOnlyList<AutomationPeer> GetOrCreateChildrenCore()
             return result;
         }
         protected override string? GetHelpTextCore()
-        {          
-            var result = AutomationProperties.GetHelpText(Owner);
+        {
+            var result = AutomationProperties.GetHelpText(Owner)
+                         ?? ToolTip.GetTip(Owner) as string;
```

Comment:
```suggestion
        {
            var result = AutomationProperties.GetHelpText(Owner);

            if (string.IsNullOrWhiteSpace(result))
            {
                result = ToolTip.GetTip(Owner) as string;
            }
```

@longbuibao I meant this change.
Since both GetHelpText and GetTip can return empty string, it should be checked twice after each. Before we use DataValidationErrors.GetErrors.